cardano-c/examples/README.md

```diff
@@ -0,0 +1,45 @@
+# Creating a wallet
+
+This example shows how to create a wallet from english mnemonics
+
+```C
+const char *alias = "Awesome Wallet";
+char *MNEMONICS = "crowd captain hungry tray powder motor coast oppose month shed parent mystery torch resemble index";
+
+/*Retrieve entropy from mnemonics*/
+cardano_entropy entropy;
+uint32_t bytes;
+cardano_bip39_error_t entropy_rc = cardano_entropy_from_english_mnemonics(MNEMONICS, &entropy, &bytes);
```

Comment:
can you also adds the way to check the returned error ? (I am happy with `assert(a == b);`)